### PR TITLE
Update to `jupyterlite-core==0.1.0`, require Python 3.8

### PR DESCRIPTION
**Pull request by jtpio**
Couple of maintenance tasks:

- Update to the latest release: https://github.com/jupyterlite/jupyterlite/releases/tag/v0.1.0
- Require Python 3.8
- Streamline the ReadTheDocs preview

### CODE REVIEW

**Comment by jtpio** [comment]
A follow-up to this would be to make a new release, and then package `jupyterlite-sphinx` on conda forge.

File: dev-environment.yml [hunk 1]
```diff
@@ -10,5 +10,4 @@ dependencies:
     - sphinx
     - black
     - pip:
-        - jupyterlite-core==0.1.0b19
-
+        - jupyterlite-core==0.1.0
```
Comment (martinRenou) [inline dev-environment.yml:13]: Maybe we could install `jupyterlite-core` with mamba instead of pip?

**Review by martinRenou — COMMENTED** [review]
Thanks!

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**